tap-xunit/README.md

```diff
@@ -0,0 +1,47 @@
+# tap-xunit
+
+TAP to xUnit XML converter.
+
+## Maintainers
+
+* The Habitat Maintainers <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+Install the package
+
+```
+hab pkg install core/tap-xunit
+```
+
+Convert your test output!
+
+```
+echo "1..4
+ok 1 - Input file opened
+not ok 2 - First line of the input valid
+ok 3 - Read the rest of the file
+not ok 4 - Summarized correctly # TODO Not written yet" | hab pkg exec core/tap-xunit tap-xunit
```

Comment:
This does not work unless you also pkg install node

```
[6][default:/src:0]# echo "1..4
ok 1 - Input file opened
not ok 2 - First line of the input valid
ok 3 - Read the rest of the file
not ok 4 - Summarized correctly # TODO Not written yet" | tap-xunit
/hab/pkgs/core/coreutils/8.30/20190115012313/bin/env: 'node': No such file or directory
[7][default:/src:127]#
```

After `hab pkg install -b core/node` it works exactly as this README describes.

Reply:
Correction - these instructions work fine. The instructions in the PR description are slightly different and do not use `hab pkg exec`.